src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2029,6 +2033,10 @@ private static bool IsUserDefinedTrueOrFalse(BoundUnaryOperator @operator)
                     pattern.InputType, pattern.NarrowedType, nullability: default, convertedNullability: default,
                     Compilation.Conversions.ClassifyBuiltInConversion(pattern.InputType, pattern.NarrowedType, ref discardedUseSiteInfo));
             }
+            if (lowestBoundNode is BoundPropertySubpatternMember member)
+            {
+                return new CSharpTypeInfo(member.Type, member.Type, nullability: default, convertedNullability: default, Conversion.Identity);
```

Comment:
I guess we want to return the unwrapped type + nullable conversion if this is a nested member? If so, we need to capture this info from binding first because it depends on the parent node. Also I'm not sure what is the expected NullabilityInfo here.

Reply:
Sound like GetTypeInfo on subpattern name wasn't handled before or am I missing something?